openadapt/tools/layout.py

```diff
@@ -0,0 +1,32 @@
+from transformers import pipeline
+from PIL import Image
+from typing import Optional
+import numpy as np
+
+
+class LayoutExtractionTool:
+    def document_query(
```

Comment:
no data is being stored within the class and so the class has no attributes, just a directly callable method.

Reply:
No class needed, can you please make this a function?